homeassistant/components/sensor/nut.py

```diff
@@ -32,6 +32,19 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
 
 SENSOR_TYPES = {
+    'device.model': ['Device model', '', 'mdi:information-outline'],
+    'device.mfr': ['Device manufacturer', '', 'mdi:information-outline'],
+    'device.serial': ['Device serial', '', 'mdi:information-outline'],
+    'device.type': ['Device type', '', 'mdi:information-outline'],
+    'device.description':
+        ['Device description', '', 'mdi:information-outline'],
+    'device.contact':
+        ['Device administrator name', '', 'mdi:information-outline'],
+    'device.location':
+        ['Device physical location', '', 'mdi:information-outline'],
+    'device.part': ['Device part number', '', 'mdi:information-outline'],
+    'device.macaddr':
+        ['Device network address', '', 'mdi:information-outline'],
```

Comment:
All of these above are static information about the device and should not be added as sensors. We will probably soon implement a "device" concept related to each entity where this type of information can be stored. But we shouldn't add it to the state machine as entity states at this point.

See https://github.com/home-assistant/architecture/issues/36.

Reply:
Makes sense. Will remove.